webapp/src/modules/externalLinks.js

```diff
@@ -1,8 +1,21 @@
 const externalLinks = {
   safeHosts: ["https://mysuma.org"],
   mobilityInfoLink: "https://mysuma.org/sumaplatform#mobility",
-  sumaIntroLink: "https://mysuma.org/",
   privacyPolicy: "https://app.mysuma.org/privacy-policy",
+  sumaIntroLink: "https://mysuma.org/",
+  edge: (lang) =>
+    `https://support.microsoft.com/${lang}-us/windows/windows-location-service-and-privacy-3a8eee0a-5b0b-dc07-eede-2a5ca1c49088`,
```

Comment:
Is `es-us` a language?

It's not- this link happens to work because MS redirects it. In fact the entire URL changes. We should be depending on broken links.